### DISCUSSION TITLE
Not overlay two links connected between elements

### DISCUSSION THREAD

**Discussion by malomiquel** [clientIO/joint#1904] [Q&A]
Hello,

I would like to know how to make sure that two (or more) links do not overlap when connecting the same two elements. On my example they will connect in the middle each time. And connect them perpendicular to the elements. 

I use boundary for the defaultConnectionPoint. And manhattan for the router of links. Because I want to connect links in the middle of each elements except one.

**Comment by kumilingus** [comment]
You have to set an element [anchor](https://resources.jointjs.com/docs/jointjs/v3.6/joint.html#anchors.intro) on the links. i.e. specify the point on the element, that the link is trying to reach. For each link, you'd set a different anchor position.

However, even this does not ensure that the links do not overlap further down the link. There is currently no automatic router in *JointJS* that would prevent the links from overlapping.
